lib/Dialect/FIRRTL/Transforms/InferResets.cpp

```diff
@@ -439,6 +439,8 @@ struct InferResetsPass : public InferResetsBase<InferResetsPass> {
   LogicalResult implementAsyncReset(FModuleOp module, ResetDomain &domain);
   void implementAsyncReset(Operation *op, FModuleOp module, Value actualReset);
 
+  LogicalResult assertOnAbstractReset();
```

Comment:
Maybe `assertHasNoAbstractReset` or `assertHasNoAbstractResetPorts(FModule module)`. Otherwise maybe it should be `abortOnAbstractReset()` or something?

Reply:
How about `verifyNoAbstractReset()` since the code doesn't contain an `assert` or aborts in any way, but produces a proper diagnostic and returns gracefully?

Just random nitpicking 😁

Reply:
Opting for Fabian's suggestion, though I'm totally fine with anything here.